docs/confluence-upgrade.md

```diff
@@ -1,8 +1,8 @@
 ## Confluence Rolling Upgrade
-Suppose we have a Confluence cluster with 3 nodes, all running `Confluence xx.xx.0-jdk11`, and an upgrade to 
-`Confluence xx.xx.1-jdk11` is planned. Use these steps to complete the upgrade process: 
+Suppose we have a Confluence cluster with 3 nodes, all running `Confluence xx.xx.0-jdk11`, and an upgrade to
```

Comment:
Are these instructions ZDU specific? If thats the case should we call out that ZDU can only be performed on patch versions (at least for Confluence) ?

Reply:
The scope of this PR was Jira so I won't be able to modify Confluence docs further. I just wanted to fix the leftover from copying the file from Bitbucket's docs (and some whitespace). Should I revert these changes to avoid confusion?